lib/src/chart/bar_chart/bar_chart_data.dart

```diff
@@ -235,7 +235,7 @@ class BarChartRodData {
     BorderRadius borderRadius,
     this.backDrawRodData = const BackgroundBarChartRodData(),
     this.rodStackItem = const [],
-  }) :borderRadius = normalizeBorderRadius(borderRadius, width);
+  }) : borderRadius = normalizeBorderRadius(borderRadius, width);
```

Comment:
Please prevent to left this kind of changes,
they just make it hard to review.
If you want to reformat somewhere, make a separate PR.
We shouldn't do reformat inside our feature request.

Reply:
My apologies. Didn't realize this got changed. 
It must have been automatically formatted by VS Code. I'll remove the space if you want.